src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+            for (int i = 0; i < 3; i++) {
+                Ball ball = Lottery.getRandomBall();
```

Comment:
The `getRandomBall()` method in the `Lottery` class is currently static. According to the checklist, static methods are generally considered bad practice. It's better to create an instance of the `Lottery` class and call the method on that instance.